src/vivarium_gates_nutrition_optimization/model_specifications/branches/scenarios.yaml

```diff
@@ -7,6 +7,6 @@ branches:
         - 'zero_coverage'
         - 'baseline'
         - 'mms'
-        - 'ifa'
+        #- 'ifa'
```

Comment:
Why did we comment this out? If just for V&V I think that's fine, but we should include this scenario in production runs!

Reply:
Hmm I'm not sure why we did this - but yes, will include it in production runs!